jbpm-services/jbpm-services-ejb/jbpm-services-ejb-timer/src/test/java/org/jbpm/services/ejb/timer/EJBTimerSchedulerTest.java

```diff
@@ -54,6 +54,7 @@ public void setup() {
         timers.add(timer1);
         
         scheduler = new EJBTimerScheduler();
+        scheduler.setUseLocalCache(true);
```

Comment:
why is this needed now?

Reply:
cache was causing a memory leak and it is not needed anymore.